corehq/apps/reports/generic.py

```diff
@@ -65,14 +65,19 @@ def _sanitize_col(col):
     return col
 
 
+def get_filter_class(field, failhard=False):
+    if isinstance(field, str):
+        klass = to_function(field, failhard=failhard)
+    else:
+        klass = field
+    return klass
```

Comment:
I'm assuming that `field` is currently only either a string or a class? If so, the parameter name `field` seems a bit misleading. Just making sure that we don't need to catch the possibility of `field` being something else?

Reply:
it's called `field` in the method it was extracted from. It only has that one use case, and the use case in our debug tools. no we don't need to catch other possibilities